app/views/users/index.html.erb

```diff
@@ -1,6 +1,6 @@
-<h1><%= current_user.name %> <%= current_user.roles[0].role %></h1>
 <% if current_user.teacher? %>
   <%= render 'index_teacher' %>
 <% else %>
-  <h1>All Blogs</h1>
+  <% # TODO: display something meaningful for student user list. %>
+  <h1>Coming Soon</h1>
```

Comment:
Is there a link to this page in the student UI? If so, might want to turn it off until this page is ready. Maybe the right thing to do is to modify the Pundit UserPolicy to not allow non-teachers until we get this page set up.